CHANGELOG.md

```diff
@@ -1,3 +1,50 @@
+# 0.0.110 - 2022-XXX
+
+## API Updates
+ * `ChannelManager::send_probe` and `Score::probe_{failed,successful}` have
+   been added to make probing more explicit, as well as new
+   `Event::Probe{Failed,Successful}` events (#1567).
+ * `ProbabilisticScoringParameters::banned_nodes` has been renamed
+   `manual_node_penalties` and changed to take msat penalties (#1592).
+ * Per-payment tracking of failed paths was added to enable configuration of
+   `ProbabilisticScoringParameters::considered_impossible_penalty_msat` (#1600)
+ * `ProbabilisticScoringParameters::base_penalty_amount_multiplier_msat` was
+   added to allow a penalty that is only amount-dependent (#1617).
+ * `ProbabilisticScoringParameters::amount_penalty_multiplier_msat` was renamed
+   `liquidity_penalty_amount_multiplier_msat` (#1617).
+ * Support has been added for DNS hostnames in the `NetAddress` type, as
+   described in BOLT PR 911 (#1553).
```

Comment:
Maybe link the BOLT PR here? Also, do we generally want to start linking the referenced issues?

```suggestion
   described in [BOLT PR #911](https://github.com/lightning/bolts/pull/911) (#1553).
```